### PR TITLE
[DEVHUB-1904] Add books page

### PR DESCRIPTION
**Pull request by mcolella14**
## Jira Ticket:

[DEVHUB-1904](https://jira.mongodb.org/browse/DEVHUB-1904)

## Description:

This is a quick MVP, so I will refactor when we expand and improve upon this.

### CODE REVIEW

**Review by harikakotipalli — APPROVED** [review]

File: src/pages/books.tsx [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+            title: 'Practical MongoDB Aggregations',
+            authors: ['Paul Done'],
+            publishedDate: 'Sept. 2023',
+            description: `This technical guide takes you on a data-driven journey by teaching you how to streamline data manipulation, resolve data processing bottlenecks, and optimize
+        pipelines. This book is your go-to resource for becoming proficient with the MongoDB
+        aggregation framework.`,
+            amazonLink:
+                'https://www.amazon.com/Practical-MongoDB-Aggregations-developing-aggregation/dp/1835080642/ref=tmm_pap_swatch_0?_encoding=UTF8&amp&qid=1694163751&amp&sr=8-3',
+            packtLink:
+                'https://www.amazon.com/Practical-MongoDB-Aggregations-developing-aggregation/dp/1835080642/ref=tmm_pap_swatch_0?_encoding=UTF8&amp&qid=1694163751&amp&sr=8-3',
```
Comment (jackyw2018) [inline src/pages/books.tsx:323]: Will this be replaced with a packtlink?

File: src/pages/books.tsx [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+                'Rachelle Palmer',
+            ],
+            publishedDate: 'Sept. 2023',
+            description: `This technical guide takes you on a data-driven journey by teaching you how to streamline data manipulation, resolve data processing bottlenecks, and optimize
+        pipelines. This book is your go-to resource for becoming proficient with the MongoDB
+        aggregation framework.`,
+            amazonLink:
+                'https://www.amazon.com/Practical-MongoDB-Aggregations-developing-aggregation/dp/1835080642/ref=tmm_pap_swatch_0?_encoding=UTF8&amp&qid=1694163751&amp&sr=8-3',
+            packtLink:
+                'https://www.amazon.com/Practical-MongoDB-Aggregations-developing-aggregation/dp/1835080642/ref=tmm_pap_swatch_0?_encoding=UTF8&amp&qid=1694163751&amp&sr=8-3',
```
Comment (jackyw2018) [inline src/pages/books.tsx:345]: Will this be replaced with a packtlink?

**Review by jackyw2018 — APPROVED** [review]
LGTM with the comment!

File: src/pages/books.tsx (hunk 1 above)
Comment (mcolella14) [reply src/pages/books.tsx:331]: Yes, I don't have some of the final copy yet, so think will change.

File: src/pages/books.tsx [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+const FeaturedBookCard: React.FunctionComponent<FeaturedBook> = ({
+    title,
+    authors,
+    thumbnail,
+    publishedDate,
+    description,
+    amazonLink,
+    packtLink,
+}) => (
+    <div
```
Comment (harikakotipalli) [inline src/pages/books.tsx:117]: No SEO section ?

Comment (mcolella14) [reply src/pages/books.tsx:117]: ah right, there was nothing specified so i left it out but I will put something in.

**Pull request merged**